### ISSUE TITLE
Dont work whitch django-tenants 3.4.4 

### ISSUE THREAD

**Issue by cristianosavoia** [fabiocaccamo/django-admin-interface#170]
**Python version**
3.10

**Django version**
4.1

**Package version**
django-admin-interface 0.20.0

**Current behavior (bug description)**
after install django-tenants 3.4.4 no migrate the error is: 

</body>
</html>

![image](https://user-images.githubusercontent.com/101615085/188102479-03f73ec8-8407-4116-b3d2-0e0433290a0a.png)
?

**Comment by fabiocaccamo** [comment]
@cristianosavoia what do you mean by "no migrate"? 
If you don't migrate you don't have the db tables ...

**Comment by cristianosavoia** [comment]
I have the tables everything ok, it was working perfect, however when installing django-tenants I do the migrate the error happens because of the admin interface, if I remove the adim inteface everything is ok. error this must be an incompatibility with the django-tenant module?

**Comment by cristianosavoia** [comment]
the error happens when I make the migrate prevent continue

**Comment by fabiocaccamo** [comment]
@cristianosavoia why you closed this? Did you find a solution?

**Comment by cristianosavoia** [comment]
no, sorry unintentionally. reopen 
Would you help me?

**Comment by merwok** [comment]
> when I make the migrate prevent continue

We can’t understand what this means, can you rephrase?

**Comment by cristianosavoia** [comment]
Using admin interference normally occurs after installing django-tenants the error shown above occurs. It seems to me that there is some bug to use the two together.

**Comment by merwok** [comment]
Can you publish a demo somewhere that reproduces the code?
Otherwise it’s hard to help here.

**Comment by fabiocaccamo** [comment]
@cristianosavoia any feedback?

**Comment by fabiocaccamo** [comment]
@cristianosavoia the message in screenshot you provided tells about a missing relation, and since this library model doesn't have any relation, I'm pretty sure that `django-tenants` creates relations between the tenant model and all other models.

If you have installed `django-admin-interface` after you already had `django-tenants` installed and configured, probably you should also run `django-tenants`'s migrations or its own command(s) for generating/managing data-schemas.

I close this issue because it's stale and also because I don't think that this problem you have can be solved on this side of the code, if you don't solve it with the suggestion above (but if you do please tell me that I'm curious to know), you should open an issue on `django-tenants` repository.




even in the cause this issue is real, it's a `django-tenants`'s issue and nothing can be done on this side.

**Comment by fabiocaccamo** [comment]
@cristianosavoia could you check if this has been fixed in `0.22.2` version?
